SwC_python_sessions_ipynb/SwC_python_session-2-2.ipynb

```diff
@@ -117,11 +117,11 @@
     "\n",
     "Like functions, which are usable parts of a program, packages (also known as libraries) are reusable programs with several modules.\n",
     "\n",
-    "Many powerful tools are built into Python which can be imported without rewriting (or even reading) them in the current program.\n",
+    "There are many powerful tools built into Python, which can be imported without rewriting (or even reading) them in the current program.\n",
```

Comment:
Ah, I see the ambiguity in the sentence. I meant that there are in-built functions that can be imported without explicitly calling them via "import" command.